packages/wdio-browserstack-service/src/service.js

```diff
@@ -27,8 +29,11 @@ export default class BrowserstackService {
     }
 
     afterTest(test) {
+        this.fullTitle = test.parent + ' - ' + test.title
```

Comment:
Does this work in Jasmine too?

Reply:
For that matter, when i finally finish the cucumber port, is this gonna block me?

Reply:
Hi,
I didn't check it with jasmine, I can do few checks and improve my code